Asterisk.2013/Asterisk.NET/Helper.cs

```diff
@@ -754,6 +754,10 @@ internal static ManagerResponse BuildResponse(Dictionary<string, string> attribu
             int hash, hashEvent;
 
             eventType = attributes["event"].ToLower(CultureInfo);
+            if (eventType == "queuememberpause") //temporary fix for Asterisk 13 pause QueueMemberPause

```

Comment:
What about version 13 is different to the others? Would be good to have a little more background here to so know why this temp fix is required, and know what to do moving forwards.